rest_api/limsdb/queries/factories.py

```diff
@@ -2,15 +2,17 @@
 from egcg_core.clarity import sanitize_user_id
 from flask import request
 from config import project_status as status_cfg
-from datetime import datetime, timedelta
 from rest_api.common import retrieve_args
 from rest_api.limsdb import queries
-from rest_api.limsdb.queries.data_models import ProjectInfo, SampleInfo, Container, Run, Sample, Project
+from rest_api.limsdb.queries import data_models
+from string import ascii_uppercase
+
+y_coords = ascii_uppercase[:8]
```

Comment:
Out of curiosity, why is this clipped to 8 characters?

Reply:
Because the 96-well plate is 12x8, so only has 8 y coordinates

Reply:
wouldn't `list('ABCDEFGH')` be more explicit? And it does not require any import